docs/install/jetson.md

```diff
@@ -1,5 +1,26 @@
 # Install on NVIDIA Jetson
 
+## Overview
+
+Jetson is NVIDIA’s line of compact, power-efficient modules designed to run AI and deep learning workloads at the edge. They combine a GPU, CPU, and neural accelerators on a single board, making them ideal for robotics, drones, smart cameras, and other embedded applications where you need real-time computer vision or inference without a cloud connection. For more details, see NVIDIA’s official Jetson overview:  
+https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/
+
```

Comment:
I recommend adding a transition line here, like "Below, we are going to walk through how to install Inference on an NVIDIA Jetson."